src/hooks/billing/useBillingRecord.ts

```diff
@@ -0,0 +1,33 @@
+import { BillingRecord, getBillingRecord } from 'api/billing';
+import { useSelectNew } from 'hooks/supabase-swr/hooks/useSelect';
+import { useBilling_selectedTenant } from 'stores/Billing/hooks';
+import { hasLength } from 'utils/misc-utils';
+
+const INTERVAL = 30000;
```

Comment:
Please make a shareable `const` for this. Since we keep using 30,000 as a default we might as well keep it in a single place.